keyboards/keycapsss/plaid_pad/keymaps/via/vial.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
Remove this file.

Reply:
Can we keep this? It has no negative effect.
Then it easier to build firmware on the vial fork.

Reply:
If VIAL is upstreamed to QMK master we may consider it, but at this stage, no.

Reply:
Done removed it.